app/main.py

```diff
@@ -1,2 +1,69 @@
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.nodes = [[]] * 8
```

Comment:
When initializing `self.nodes`, you're using:

```python
self.nodes = [[]] * 8
```

This creates 8 references to the same list, so changing one affects all others. Instead, you should use:

```python
self.nodes = [[] for _ in range(8)]
```

This ensures each bucket is a separate, distinct list.